examples/lambda-layer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../cdk.schema.json",
+  "Resources": {
+    "AwsCliLayer": {
+      "Type": "aws-cdk-lib.lambda_layer_awscli.AwsCliLayer"
+    },
+    "Lambda": {
```

Comment:
Minor point: is it possible to demonstrate this by only declaring the layer? If so, the generated template would be shorter and easier to read and see the point.